woocommerce/woocommerce-gutenberg-products-block
# Add several tests to Checkout block in the editor

Adding some more tests to the Checkout block.

### How to test the changes in this Pull Request:

[Run e2e tests](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/main/docs/contributors/javascript-testing.md#how-to-run-end-to-end-tests) and verify they pass.
